src/Selenium2Driver.php

```diff
@@ -222,25 +222,29 @@ protected function withSyn()
     /**
      * Creates some options for key events
      *
-     * @param string $char     the character or code
-     * @param string $modifier one of 'shift', 'alt', 'ctrl' or 'meta'
+     * @param string|int  $char     the character or code
+     * @param string|null $modifier one of 'shift', 'alt', 'ctrl' or 'meta'
      *
      * @return string a json encoded options array for Syn
      */
     protected static function charToOptions($char, $modifier = null)
     {
-        $ord = ord($char);
-        if (is_numeric($char)) {
-            $ord = $char;
+        if (is_int($char)) {
+            $charCode = $char;
+            $char = chr($charCode);
+        } else {
+            $charCode = ord($char);
         }
```

Comment:
Please revert this code fragment, because it could potentially lead to the BC break when `is_int($char)` and `is_numeric($char)` yields different results.

Reply:
@aik099 for ex. when `1` char/string is passed, I expect `1` to be pressed, `is_numeric` was wrong here, also all tests pass

Reply:
That's an interesting observation. Never thought that number (e.g. `1`, `7`) could also be interpreted as a symbol and not a char code. Need to look closely at these tests you're talking about.

Reply:
@aik099 please merge https://github.com/minkphp/driver-testsuite/pull/61 test for it, also adds a test for capital letter key press

Reply:
@aik099 can you please merge the test PR and this PR then?

Reply:
@stof , I've debugged the original (prior to this PR) version of the `\Behat\Mink\Driver\Selenium2Driver::charToOptions` method and the results are as follows:

* When a letter is given as input, then it's converted into an ASCII code
* When a number is given (as a string or as an integer) it's interpreted as an ASCII code itself

This behavior partially matches the DocBlock on the `\Behat\Mink\Driver\DriverInterface::keyPress`:

```
* @param string|int $char     could be either char ('b') or char-code (98)
 ```

Due to the usage of the `is_numeric` function (in implementation), a digit from `'0'` to `'9'` passed in as a string was interpreted as an ASCII code as well, which is a bug.

To overcome this developers likely had to pass in the ASCII code of these digits. Tests for the such case were missing, but now they're added via https://github.com/minkphp/driver-testsuite/pull/61 .